frontend/src/components/FAQHelp/HelpSearchBar.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from 'react';
+
+const SearchBar = (): ReactElement => {
+  return <div className="faq-help-search input-group mb-3">
+    <input type="text" className="form-control" placeholder="Search for any question!" aria-label="Search locations" aria-describedby="basic-addon2" />
+    <div className="input-group-append">
+      <button className="btn btn-light" type="button">
+        <svg width="1em" height="1em" viewBox="0 0 16 16" className="bi bi-search" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Also mentioned this in Kinjal's PR, but instead of pasting SVG data into the React frontend code, you should put it in an `.svg` file (maybe make an `assets/` folder in the frontend src) and then import it to use it as a component.